ReleaseProcess.md

```diff
@@ -0,0 +1,97 @@
+This is an early draft of the release procecss for icecc.  It will need to be refined over time.
+
+The steps to do a github release are
+
+1. Create a branch for the new release
+  1. update the version in master
```

Comment:
I think this needs clearer instructions - it's just configure.ac or more?

Reply:
Ah, you have more instructions on this later - so just reference that. Sorry I didn't read the full before adding comments